src/core/views.py

```diff
@@ -1617,6 +1658,7 @@ def edit_editorial_group(request, group_id=None):
                 else:
                     group = form.save(commit=False)
                     group.journal = request.journal
+                    group.press = request.press
```

Comment:
Sorry, I meant to make the press field required. I've done that now. If you could check the migration and make sure you agree with how I've done it, that would be great.

As for whether the form is valid, it has the same behavior as before. The journal field was already excluded from the form in this view's logic, even when it was required, so I have excluded the press as well.

The view is a bit convoluted, but it does always add the journal and press, either here, via the instance that is passed to the form...

https://github.com/BirkbeckCTP/janeway/blob/921328e81b22b21938f5b01a54de759fae348a30/src/core/views.py#L1637-L1638

...or here, after the form is saved:

https://github.com/BirkbeckCTP/janeway/blob/921328e81b22b21938f5b01a54de759fae348a30/src/core/views.py#L1660-L1661

If that's a problem, it was pre-existing, but we can refactor further...

Reply:
Right, I missed we are excluding `journal` there on the form fields. I'm not a big fan of the level of indirection here (it should be the form who sets `journal` and `press` rather than the view) but as you say, it is pre-existing code. I added one